### PR TITLE
fix(docs): additional information in docs and renamed hosts file

### PR DESCRIPTION
**Pull request by jones718**
Added information in getting_started.md and usage.md and renamed hostnames in hosts file to camel case.

### CODE REVIEW

**Review by p0wer0xff — APPROVED** [review]

File: getting_started.md [hunk 1]
```diff
@@ -62,13 +62,18 @@ The FLUFFI Utility Network (FUN) contains all the infrastructure needed for fuzz
 
 To set it up, you first need to set up the central server with docker installed. Give that server the static IP address 10.66.0.1.
 
-Copy the FLUFFI repo to that machine (or clone it there).
+Copy the FLUFFI repo to that machine (or clone it there). For git clone, make sure to have [git lfs](https://git-lfs.github.com/) installed.
```
Comment (maverickwoo) [inline getting_started.md]: Some folks including myself not long ago may think that `apt install git-lfs` (or equivalent) is sufficient. Unfortunately, it is not and the user likely needs to run `git lfs install` to install the hooks before cloning.

My suggestion is to say:
> ... installed and then properly configured using `git lfs install`.

File: getting_started.md [hunk 2]
```diff
@@ -84,7 +89,7 @@ Apply changes to the default configuration. We recommend you to:
 - change /srv/fluffi/data/smb/files/initial/MAC2Host.csv to contain the MAC addresses and hosts of your executor machines (if you want to set host names according to MAC addresses)
 - replace the files in /srv/fluffi/data/smb/files/initial/updatesWS2008 and /srv/fluffi/data/ftp/files/initial/windows/ansible with the actual files. The files stored there in the git are just text files with links to where you can download the actual files.
 
-Copy the GlobalManager to /srv/fluffi/data/fluffigm
+Copy the GlobalManager to /srv/fluffi/data/fluffigm (make sure to include *.so files)
```
Comment (maverickwoo) [inline getting_started.md]: This instruction can still be a bit vague. "GlobalManager" here refers to the binary `core/{{arch}}/bin/GlobalManager` and the `.so` files refers to the `lib*.so*` files in the same directory. We can make it explicit by saying:

> Copy `core/{{arch}}/bin/{GlobalManager,lib*.so*}` to `/srv/fluffi/data/fluffigm`.

File: getting_started.md [hunk 3]
```diff
@@ -96,18 +101,25 @@ find /srv/fluffi/data/smb/files -type d -exec chmod 777 {} \;
...
+For the following two steps, an internet connection is necessary at the first time.
 Get FLUFFI's web dependencies:
 ```
-cd srv/fluffi/data/fluffiweb/app
+cd /srv/fluffi/data/fluffiweb/app
 ./get_static_dependencies.sh
 ```
 
-Finally, start all server services
+Finally, build all server services: (make sure to stop dns daemon before that)
```
Comment (maverickwoo) [inline getting_started.md]: I suggest saying:
> stop the DNS daemon on the host before that, e.g., `sudo systemctl stop systemd-resolved` if using Ubuntu.

**Review by maverickwoo — COMMENTED** [review]
I think this PR hits all the high notes. Thanks!

(I suggest we iteratively improve the doc over time and so please feel free to merge.)

File: getting_started.md (hunk 3 above)
Comment (p0wer0xff) [reply getting_started.md]: oh, you have a good point there - another part missing in the docs. We replace some of systemd's more intrusive and interfering stuff on all of our machines:
```bash
#!/bin/bash

systemctl stop systemd-networkd.service
systemctl stop systemd-resolved.service
systemctl disable systemd-networkd.service
systemctl disable systemd-resolved.service
systemctl mask systemd-networkd.service
systemctl mask systemd-resolved.service

rm -f /etc/resolv.conf
echo "search fluffi" > /etc/resolv.conf
echo "nameserver 10.66.0.1" >> /etc/resolv.conf

mv /etc/network/interfaces /etc/network/interfaces.unused

DEBIAN_FRONTEND=noninteractive apt-get install -yq install network-manager

sed ':a;N;$!ba;s/\[main\]\n/[main]\ndns=default\n/g' /etc/NetworkManager/NetworkManager.conf
sed 's/renderer: networkd/renderer: NetworkManager/g' /etc/netplan/01-netcfg.yaml
netplan apply
systemctl restart network-manager
```
this is roughly the same script as [the one that runs on first install](https://github.com/siemens/fluffi/blob/master/srv/fluffi/data/smb/files/initial/initialConfiguration.sh#L108) on the agent machines, but it's not automated as that one - we just ran it manually on the infrastructure boxes. @jones718 - could you please and add this to the docs?

**Pull request merged**